connect-inject/container_init.go

```diff
@@ -458,7 +453,7 @@ chmod 444 /consul/connect-inject/acl-token
   /consul/connect-inject/service.hcl
 
 # Generate the envoy bootstrap code
-/bin/consul connect envoy \
+/consul/connect-inject/consul connect envoy \
```

Comment:
Since we are now using consul-k8s image instead of consul the path will be from the shared vol instead of `/bin/`.. update all the tests accordingly.